### DISCUSSION TITLE
How to Hide The Footer

### DISCUSSION THREAD

**Discussion by ghost** [ArchiveBox/ArchiveBox#769] [Q&A]
Is there is an option to completely hide the footer in the saved web sites? 
it bothers me..
![14](https://user-images.githubusercontent.com/18482745/121636107-42a6ac80-ca3c-11eb-8c22-be5f3e01e804.png)

**Comment by pirate** [comment]
Do you mean header? If so, just click on the filename to open it without the header: 
<img width="1179" alt="Screen Shot 2021-06-11 at 2 04 39 a" src="https://user-images.githubusercontent.com/511499/121638747-7f34d100-ca59-11eb-83d0-8bf401bf9e0b.png">

We can maybe default to having the header start collapsed in the future, but the default UI would need to make it clear that info is there if needed, and that will take some UX design work to get right without making it look cluttered.
